public/css/Add-Note-context-menu.css

```diff
@@ -0,0 +1,263 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Times New Roman', Times, serif;
+}
+
+aside {
+    float: left;
+    height: 100%;
+}
+
+nav {
+    height: 3rem;
+    display: grid;
+    justify-content: end;
+    align-items: center;
+    padding: 0 3rem;
+}
+
+p {
+    text-align: justify;
+}
+
+textarea {
+    resize: none;
+}
+
+textarea:focus,
+input:focus {
+    outline: none;
+}
+
+#add-card {
+    display: grid;
+    justify-content: center;
+}
+
+#add-card-button {
+    height: 2.3rem;
+}
+
+#create-note {
+    margin: 3rem auto;
+
+    width: 32rem;
+}
+
+#note-details {
+    background-color: #fff;
+
+    border: solid 1px #E0E0E0;
+    border-radius: 0.625rem;
+
+    width: 40rem;
+
+    margin: auto;
+    padding: 1.25rem;
+
+    display: flex;
+    flex-direction: column;
+    max-height: 75vh;
+
+}
+
+.add-card__details {
+
+    background-color: #fff;
+
+    border: solid 1px #E0E0E0;
+    border-radius: 0.3rem;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+
+    text-align: left;
+    padding-left: 0.8rem;
+}
+
+.button-details {
+    margin: 0.3rem 0;
+
+    height: 2rem;
+    width: 4rem;
+
+    background-color: #fff;
+
+    border: none;
+}
+
+.button-details:hover,
+.note-content__footer-button:hover {
+    background-color: #f5f5f5;
+
+    border-radius: 0.3rem;
+}
+
+.create-note__details {
+    border: none;
+
+    width: 100%;
+
+    margin: 0.625rem 0;
+}
+
+.list-grid__button {
+    padding: 0.5rem;
+    background-color: white;
+    border: none;
+}
+
+.list-grid__button:hover {
+    background-color: #E9EAEA;
+
+    border: 1px solid #E9EAEA;
+    border-radius: 100%;
+}
+
+.list-grid__button[data-view="list"] [data-icon="grid"] {
+    display: none;
+}
+
+.list-grid__button[data-view="grid"] [data-icon="list"] {
+    display: none;
+}
+
+.note {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+
+    margin: 0.8rem 2rem;
+    padding: 1rem;
+
+    border: solid 1px #E0E0E0;
+    border-radius: 0.625rem;
+}
+
+.note__header,
+.note__footer,
+menu {
+    display: grid;
+    grid-template-columns: 1fr auto;
+
+    align-items: center;
+    justify-content: space-between;
+}
+
```

Comment:
Add 

.note__footer {
    height: 1.8rem;
}

to reserve the space for icons so that we don't move up and down every element in the page.